### PR TITLE
No wait for start VM.

### PR DESCRIPTION
**Pull request by LiliDeng**
Test results
```
[LISAv2 Test Results Summary]
Test Run On           : 12/11/2019 07:41:16
ARM Image Under Test  : canonical : ubuntuserver : 18.04-lts : Latest
Initial Kernel Version: 5.0.0-1027-azure
Final Kernel Version  : 5.0.0-1027-azure
Total Test Cases      : 6 (6 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:57

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NETWORK              NETWORK-ADD-MAX-SYNTHETIC-NICS-AFTER-PROVISION-1                                  PASS                 6.24 
    2 NETWORK              NETWORK-ADD-MAX-SYNTHETIC-NICS-AFTER-PROVISION-2                                  PASS                 6.75 
    3 NETWORK              NETWORK-ADD-MAX-SYNTHETIC-NICS-AFTER-PROVISION-3                                  PASS                 7.77 
    4 SRIOV                SRIOV-ADD-MAX-NICS-AFTER-PROVISION-1                                              PASS                 6.71 
    5 SRIOV                SRIOV-ADD-MAX-NICS-AFTER-PROVISION-2                                              PASS                  6.1 
    6 SRIOV                SRIOV-ADD-MAX-NICS-AFTER-PROVISION-3                                              PASS                 6.29 
```

### CODE REVIEW

**Comment by LiliDeng** [comment]
Related with #646

File: Testscripts/Windows/NET-Add-NIC-After-Provision.ps1 [hunk 1]
```diff
@@ -22,15 +22,15 @@ function Main {
 	try {
 		$extraNICs = $TestParams.EXTRA_NICS
 		$testResult = "FAIL"
-
+		Write-LogDbg "Stop VM $($AllVMData.RoleName) in RG $($AllVMData.ResourceGroupName) - start."
```
Comment (juhlee-microsoft) [inline Testscripts/Windows/NET-Add-NIC-After-Provision.ps1]: Wording? "Stopping VM ...." and omit " - start"?

File: Testscripts/Windows/NET-Add-NIC-After-Provision.ps1 [hunk 2]
```diff
@@ -22,15 +22,15 @@ function Main {
...
 		# Stop VM before attaching new NICs
 		Stop-AzVM -ResourceGroupName $AllVMData.ResourceGroupName -Name $AllVMData.RoleName `
 			-Force | Out-Null
 		if (-not $?) {
 			Write-LogErr "Failed to stop $($AllVMData.RoleName)"
 			return "FAIL"
 		}
-
+		Write-LogDbg "Stop VM $($AllVMData.RoleName) in RG $($AllVMData.ResourceGroupName) - end."
```
Comment (juhlee-microsoft) [inline Testscripts/Windows/NET-Add-NIC-After-Provision.ps1]: "Completed VM stoppping" and omit " - end"

File: Testscripts/Windows/NET-Add-NIC-After-Provision.ps1 [hunk 3]
```diff
@@ -56,20 +56,31 @@ function Main {
 				Write-LogErr "Failed to create extra NIC #${nicNr} in $($AllVMData.ResourceGroupName)"
 				return "FAIL"
 			}
-			Start-Sleep -s 5
+			Start-Sleep -Seconds 5
 			Write-LogInfo "Successfully added extra NIC #${nicNr}!"
 		}
+		Write-LogDbg "Update VM $($AllVMData.RoleName) in RG $($AllVMData.ResourceGroupName) - start."
```
Comment (juhlee-microsoft) [inline Testscripts/Windows/NET-Add-NIC-After-Provision.ps1]: Same wording recommendation here also.

File: Testscripts/Windows/NET-Add-NIC-After-Provision.ps1 [hunk 4]
```diff
@@ -56,20 +56,31 @@ function Main {
...
 		# Start VM
-		Start-AzVM -ResourceGroupName $AllVMData.ResourceGroupName -Name $AllVMData.RoleName | Out-Null
+		Write-LogDbg "Start VM $($AllVMData.RoleName) in RG $($AllVMData.ResourceGroupName)."
+		Start-AzVM -ResourceGroupName $AllVMData.ResourceGroupName -Name $AllVMData.RoleName -NoWait | Out-Null
 		if (-not $?) {
 			Write-LogErr "Failed to start $($AllVMData.RoleName)"
 			return "FAIL"
 		}
+		$vm = Get-AzVM -ResourceGroupName $AllVMData.ResourceGroupName -Name $AllVMData.RoleName -Status
+		$MaxAttempts = 20
```
Comment (juhlee-microsoft) [inline Testscripts/Windows/NET-Add-NIC-After-Provision.ps1]: I am not sure if this works with bigger size VM provision time. Can you check 200 seconds with M416 or HB120v2 VMs?

**Review by juhlee-microsoft — CHANGES_REQUESTED** [review]

File: Testscripts/Windows/NET-Add-NIC-After-Provision.ps1 (hunk 4 above)
Comment (LiliDeng) [reply Testscripts/Windows/NET-Add-NIC-After-Provision.ps1]: I am testing against bigger size, seems we need increase the max attempts.

**Review by juhlee-microsoft — APPROVED** [review]

**Comment by LiliDeng** [comment]
Test results against bigger size, max wait time after start for size Standard_M416s_v2 is around 5-6 mins from job job-azure-cloudtest-manual/3978
```
[LISAv2 Test Results Summary]
Test Run On           : 12/12/2019 13:26:13
ARM Image Under Test  : SUSE : SLES : 12-sp4-gen2 : 2019.11.13
Override VM size      : Standard_M416s_v2
Initial Kernel Version: 4.12.14-6.29-azure
Final Kernel Version  : 4.12.14-6.29-azure
Total Test Cases      : 3 (3 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:2:59

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 NETWORK              NETWORK-ADD-MAX-SYNTHETIC-NICS-AFTER-PROVISION                                    PASS                16.38 
    2 SRIOV                SRIOV-ADD-MAX-NICS-AFTER-PROVISION                                                PASS                15.91 
    3 SRIOV                SRIOV-DISABLE-ENABLE-AN                                                           PASS               121.23 
	EnableSRIOV : Test Iteration - 1 : PASS 
	DisableSRIOV : Test Iteration - 1 : PASS 
	EnableSRIOV : Test Iteration - 2 : PASS 
	DisableSRIOV : Test Iteration - 2 : PASS 
	EnableSRIOV : Test Iteration - 3 : PASS 
	DisableSRIOV : Test Iteration - 3 : PASS 
	EnableSRIOV : Test Iteration - 4 : PASS 
	DisableSRIOV : Test Iteration - 4 : PASS 
	EnableSRIOV : Test Iteration - 5 : PASS 
	DisableSRIOV : Test Iteration - 5 : PASS 
```

**Pull request merged**